docs/community/contribute.md

```diff
@@ -24,45 +24,45 @@ Following these guidelines shows that you respect the time of the developers who
 
 ### What Can I Contribute?
 
-We are a fully Open Source project, meaning we are open to all kinds of contributions from our community. Here are a few examples of contributions that we are open to in this repo:
+We are a fully open source project, meaning we are open to all kinds of contributions from our community. Here are a few examples of contributions that we are open to:
 
-- **Added Content:** Writing a nice guide? Submit it (Where?) so we can add it to our Docs. 
-- **Translation:** Adding Translations of our prior content? See the [translate](TRANSLATE) page for help.
-- **Bug Reports & Fixes:** Find a bug or error? Let us know where it's hiding. (Where?)
+- **Added Content:** Writing a nice guide? Submit it on the [forums](https://forum.subspace.network/c/guides-and-faqs/13). 
+- **Translation:** Adding translations of our prior content? See the [translate](TRANSLATE) page for help.
+- **Bug Reports & Fixes:** Find a bug or error? Let us know where it's hiding. Report it at the [GitHub](https://github.com/subspace/subspace-docs).
```

Comment:
Sorry, I missed this one last time. The bug should be reported to the [support forum](https://forum.subspace.network/c/support/5).